action.yml

```
@@ -37,6 +37,9 @@ inputs:
   context:
     description: "Build's context is the set of files located in the specified PATH or URL"
     required: false
+  env-secrets:
+    description: "List of secrets to expose to the build using environment variables from the GitHub runner (e.g., MY_SECRET=MY_ENV_VAR)"
```

I think "using environment variable" is enough. cc @dvdksn 

```suggestion
    description: "List of secrets to expose to the build using environment variables (e.g., MY_SECRET=MY_ENV_VAR)"
```

---

agree